langchain/document_loaders/gcs_directory.py

```diff
@@ -27,6 +27,8 @@ def load(self) -> List[Document]:
         client = storage.Client(project=self.project_name)
         docs = []
         for blob in client.list_blobs(self.bucket, prefix=self.prefix):
+            if blob.name.endswith("/"):
```

Comment:
Could you add a comment in the code saying why this is needed?

Reply:
Done